app/helpers/provider_helper.rb

```diff
@@ -13,23 +13,23 @@ def visa_sponsorship_status(provider)
     if !provider.declared_visa_sponsorship?
       visa_sponsorship_call_to_action(provider)
     elsif provider.can_sponsor_student_visa || provider.can_sponsor_skilled_worker_visa
-      "You can #{visa_sponsorship_short_status(provider)}"
+      "#{visa_sponsorship_short_status(provider)} can be sponsored"
     else
-      "You cannot sponsor visas"
+      "Visas cannot be sponsored"
     end
   end
 
   def visa_sponsorship_short_status(provider)
     if !provider.declared_visa_sponsorship?
       visa_sponsorship_call_to_action(provider)
     elsif provider.can_sponsor_all_visas?
-      "sponsor Student and Skilled Worker visas"
+      "Student and Skilled Worker visas"
     elsif provider.can_only_sponsor_student_visa?
-      "sponsor Student visas"
+      "Student visas"
     elsif provider.can_only_sponsor_skilled_worker_visa?
-      "sponsor Skilled Worker visas"
+      "Skilled Worker visas"
     else
-      "sponsor visas"
+      "Visas" # TODO: drop this? Don't think we'd ever reach this condition.
```

Comment:
@davidgisbey I think the `else` condition is never reachable, but maybe it was added for a good reason? 🤷

Reply:
yea looks redundant to me